zyte_common_items/processors.py

```diff
@@ -108,7 +108,7 @@ def brand_processor(value: Union[Selector, HtmlElement], page: Any) -> Any:
 
 
 @only_handle_nodes
-def price_processor(value: Union[Selector, HtmlElement], page: Any) -> Any:
+def price_processor(value: Union[Selector, HtmlElement, str, None], page: Any) -> Any:
```

Comment:
I don't think this can work? `@only_handle_nodes` calls the wrapped function only for `Union[Selector, HtmlElement]`.